docs/_posts/2022-04-01-issue-31.md

```diff
@@ -27,6 +27,30 @@ We welcome your feedback and suggestions. Let us know if we missed anything inte
 
 *  A
 
+### HLSL Working Group
+
+* A new [HLSL Working Group](https://discourse.llvm.org/t/hlsl-working-group/61279/) is being formed to coordinate efforts for adding HLSL and related code generation support to LLVM & Clang.
+  * A living [Agenda/Meeting Notes](https://docs.google.com/document/d/1RvDN615hd9dJEBg4QqTG3emgB-1UWXQG2q2bZbdUTTo/edit?usp=sharing) document is available.
+  * Meetings are planned to be 30 minutes long bi-weekly and will adjust as appropriate.
+* Tom Stellard (tstellar), has created a #hlsl channel on the [LLVM Discord server](https://discord.gg/xS7Z362).
+
+#### DirectX Backend
+* The experimental DirectX backend stub is landed, and subsequent commits are under review.
+  * [Add DXILPrepare CodeGen pass](https://reviews.llvm.org/D122081)
```

Comment:
The format is typically:
<bullet> Change in past tense <dot> patch-link
with function names / types in verbatim.

For example:
```
  * Added `DXILPrepare` CodeGen pass. [D122081](https://reviews.llvm.org/D122081)
```